src/components/PasswordInput/PasswordInput.tsx

```diff
@@ -0,0 +1,121 @@
+import { Button, ButtonEmphasis, ButtonSize, ButtonStyle } from '..';
+
+export type PasswordInputProps = { obfuscated?: boolean } & Omit<
+    InputBaseProps<string>,
+    'valueSelect' | 'decorator' | 'suffix'
+>;
+
+export const PasswordInput = ({
+    autocomplete = false,
+    clearable,
+    disabled,
+    id: propId,
+    obfuscated = true,
+    placeholder,
+    readOnly,
+    required,
+    status = Validation.Default,
+    size,
+    value,
+    onChange,
+    onFocus,
+    onBlur,
+    onKeyDown,
+    'data-test-id': dataTestId = 'fondue-password-input',
+}: PasswordInputProps) => {
+    const inputElementRef = useRef<HTMLInputElement | null>(null);
+    const [isObfuscated, setIsObfuscated] = useState<boolean>(obfuscated);
+
+    const { isFocusVisible, focusProps } = useFocusRing({ isTextInput: true });
+
+    const handleOnChange = useCallback(() => {
+        onChange?.(inputElementRef.current?.value);
+    }, [inputElementRef, onChange]);
+
+    const handleClear = () => {
+        if (inputElementRef.current) {
+            inputElementRef.current.value = '';
+            handleOnChange();
+        }
+    };
+
+    return (
+        <div
+            className={merge([
+                'tw-flex tw-items-center tw-justify-between tw-h-9 tw-gap-2 tw-px-3 tw-py-3 tw-transition tw-text-s tw-font-sans tw-relative dark:tw-bg-transparent tw-border tw-rounded',
+                size ? '' : 'tw-w-full',
+                disabled
+                    ? 'tw-bg-box-disabled hover:tw-cursor-not-allowed tw-border-line-weak'
+                    : 'tw-bg-base tw-border-line-strong hover:tw-border-line-x-strong focus-within:tw-border-line-xx-strong focus-within:hover:tw-border-line-xx-strong',
+                isFocusVisible && FOCUS_STYLE,
+                status ? validationClassMap[status] : '',
+            ])}
+            data-test-id={dataTestId}
+        >
+            <input
+                size={size}
+                value={value}
+                onBlur={onBlur}
+                onFocus={onFocus}
+                disabled={disabled}
+                readOnly={readOnly}
+                required={required}
+                ref={inputElementRef}
+                onKeyDown={onKeyDown}
+                onChange={handleOnChange}
+                placeholder={placeholder}
+                id={useMemoizedId(propId)}
+                aria-label="Password Input"
+                autoComplete={autocomplete ? 'on' : 'off'}
+                onClick={() => inputElementRef.current?.focus()}
+                type={isObfuscated ? InputTypes.Password : InputTypes.Text}
+                className={merge([
+                    'tw-w-full tw-border-none tw-outline-none tw-bg-transparent tw-hide-input-arrows tw-text-sm',
+                    disabled || readOnly
+                        ? 'tw-text-text-disabled tw-placeholder-text-disabled dark:tw-text-black-30 dark:tw-placeholder-black-40 hover:tw-cursor-not-allowed'
```

Comment:
We need to sync with the Designers as it still doesn't look good on dark mode. 
We may also need to align on the alias tokens to be used here:
![Screenshot 2023-11-07 at 16 01 58](https://github.com/Frontify/fondue/assets/35405438/dee6b880-d6b1-4690-946c-f3db9a03f70a)
![Screenshot 2023-11-07 at 16 01 48](https://github.com/Frontify/fondue/assets/35405438/321fc00b-f6e9-418e-8e5d-8c9118bab7be)

Reply:
I will raise this in the channel

Reply:
> We need to sync with the Designers as it still doesn't look good on dark mode. We may also need to align on the alias tokens to be used here:
> 
> Otherwise, it looks really good. Great work!! 🎉

@imoutaharik tokens have be confirmed and are updated.

Reply:
Thanks, It looks much better on dark mode, especially for the text color. But on dark mode, validation doesn't change the border color.

Reply:
@imoutaharik good spot.  The validation map in general was missing `dark` theme presets.  This has been updated